### ISSUE TITLE
launchctl configuration out-of-date 

### ISSUE THREAD

**Issue by akloeber** [syncany/syncany#488]
When `homebrew.mxcl.syncany.plist` is loaded into launchctl as documented by the hombre formula output the following error is logged:
```
Error: Invalid syntax, no action given (start, stop, reload, restart, status, force-stop, list, add, remove).
       Refer to help page using '--help'.
```

The same error appears if `sy --daemon` is run manually. I suppose `homebrew.mxcl.syncany.plist` has to be adjusted in order to invoke `sy daemon start` instead.

__Environment:__
Syncany 0.4.5-alpha
Mac OS X 10.10.3

**Comment by cr0** [comment]
Thanks for reporting!

@binwiederhier Did we change the syntax or what?

**Comment by binwiederhier** [comment]
@cr0 I can't remember that we ever had `sy --daemon`. `sy daemon start` is definitely right.

**Comment by pimotte** [comment]
Fixed by #530
